src/blog/jwt-identification-and-hot-streams/index.md

```diff
@@ -0,0 +1,173 @@
+# AnyCable PRO:<br>JWT identification and "hot streams"
+
+September 27, 2021
+{date}
+
+We've just launched the final [AnyCable PRO][pro] release that brings advanced features and 40% less memory footprint to AnyCable—the product to build real-time features for Ruby on Rails apps. The two brand-new PRO features we're presenting in the post, [JWT identification][jwt-id-docs] and [signed streams support][signed-streams-docs], arrived from the Early Access program. We asked its adopters to share how they use _cables_ and which features they would like us to provide out of the box. They might seem unrelated to each other at first glance, but when used together, they have the potential to bring about a huge performance boost.
+{intro}
+
+<div class="divider"></div>
+
+## Dealing with authentication
+
+Amongst all the requested features, one of the leading requests was a "token-based authentication". AnyCable reckons upon the Action Cable API (although it can work outside of Rails), and most Action Cable applications rely on cookies as an authentication mechanism (either directly or via sessions). The main benefit of cookie-based authentication is simplicity—it just works. However, there are some drawbacks:
```

Comment:
AnyCable reckons --> AnyCable relies 

@palkan it seems this was a mistake

Reply:
It's double "rely" in one sentence then. Could you please suggest a synonym for one of them?

Reply:
_I would replace the second case:_
most Action Cable applications rely on cookies --> most Action Cable applications utilize cookies

Reply:
Ok!

Reply:
```suggestion
Amongst all the requested features, one of the leading requests was a "token-based authentication". AnyCable relies on the Action Cable API (although it can work outside of Rails), and most Action Cable applications utilize cookies as an authentication mechanism (either directly or via sessions). The main benefit of cookie-based authentication is simplicity—it just works. However, there are some drawbacks:
```